File: C#/uri-1011.cs
// URI - 1011

using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            double radius = double.Parse(Console.ReadLine());

            Console.WriteLine("VOLUME = "+ ((4.0 / 3.0) * 3.14159 * Math.Pow(radius, 3)).ToString("F3"));
        }
    }
}

File: URI/JavaScript/uri-1011.js
// URI - 1011: https://www.urionlinejudge.com.br/judge/pt/problems/view/1011

const readline = require('readline')
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
})

rl.on('line', input => {
    const radius = parseInt(input)

    console.log('VOLUME = ' + ((4.0/3.0) * 3.14159 * Math.pow(radius, 3)).toFixed(3))

    rl.close()
})